src/core/fetchers/segment/segment_fetcher_creator.ts

```diff
@@ -82,20 +86,34 @@ export default class SegmentFetcherCreator {
    */
   private readonly _backoffOptions : ISegmentFetcherCreatorBackoffOptions;
 
+  private readonly _cdnPrioritizer : CdnPrioritizer;
+
   /**
    * @param {Object} transport
    */
   constructor(
     transport : ITransportPipelines,
-    options : ISegmentFetcherCreatorBackoffOptions
+    manifest : Manifest,
+    options : ISegmentFetcherCreatorBackoffOptions,
+    cancelSignal : CancellationSignal
   ) {
+    const steeringManifestFetcher = transport.steeringManifest === null ?
+      null :
+      new SteeringManifestFetcher(transport.steeringManifest,
+                                  { maxRetryOffline: undefined,
+                                    maxRetryRegular: undefined });
```

Comment:
As the steeringManifestFetcher is only used inside the cdnPrioritizer, isn't it better to create it inside the CDNPrioritizer ?

Reply:
Yes, it does seems better.

I'll see if it works well

Reply:
Done